scripts/js/lib/api/processHtml.test.ts

```diff
@@ -313,6 +314,25 @@ test("addLanguageClassToCodeBlocks()", () => {
     </div>`);
 });
 
+test("updateRedirectedExternalLinks()", () => {
+  const redirects = {
+    "https://ibm.com/old": "https://ibm.com/new",
+    "https://basename.example-website.com/old#anchor?and=query":
+      "https://newbasename.example.com/new#newanchor?query=new",
+  };
+  const doc = CheerioDoc.load(
+    `<a href="https://ibm.com/old">https://ibm.com/old</a>
+    <a href="https://basename.example-website.com/old#anchor?and=query"></a>
```

Comment:
Can you please add a URL that is not redirected? Ideally it looks very similar to an old URL, such as `https://ibm.com/old-but-not-redirected`. We want to only change exact matches.